cpp/src/arrow/python/helpers.cc

```diff
@@ -99,7 +99,8 @@ Status InferDecimalPrecisionAndScale(PyObject* python_decimal, int32_t* precisio
   DCHECK_NE(precision, NULLPTR);
   DCHECK_NE(scale, NULLPTR);
 
-  OwnedRef as_tuple(PyObject_CallMethod(python_decimal, "as_tuple", "()"));
+  // TODO(phillipc): Make sure we perform PyDecimal_Check(python_decimal) as a DCHECK
```

Comment:
This requires ARROW-2160 to address.

Reply:
Do you want to wait for ARROW-2145 or merge this independently?

Reply:
I have a couple more tests to add here